### DISCUSSION TITLE
New template for UI5WC and React based on Vite with some cool features!

### DISCUSSION THREAD

**Discussion by dewdad** [UI5/webcomponents-react#3046] [Show and tell]
Hi!

I've published a new [starter template](https://github.com/dewdad/ui5-react-vite) with the new [Vite](https://blog.bitsrc.io/vite-is-better-than-webpack-d5dd59610d56) frontend tooling framework. Also providing
- Service proxying with [TripPin](https://www.odata.org/blog/trippin-new-odata-v4-sample-service/) OData V4 service
- File-based routing
- Most of the features from the CRA seed template
- And more...

I would love to collaborate on it to grow and engage the community in size and productivity. If this could be of interest, I would love to discuss and get some guidance.

Cheers, and happy coding!

**Comment by MarcusNotheis** [comment]
Hey @dewdad,

that's really awesome, looking very promising!
In my opinion, the [ui5-web-components-templates org](https://github.com/ui5-web-components-templates) would be good place for this template to live, this org is also linked in our documentation.
Maybe @oarthursilva can give some details on how to contribute a new template to that org?
